tests/test_actions.py

```diff
@@ -82,12 +96,14 @@ def test_hides_secret_headers(self, APITester, caplog):
         assert str(test_headers) not in caplog.text
 
 
-def test_generate_send_method_class_docstring():
+def test_generate_send_method_class_docstring() -> None:
     """Generated class and method's docstring both contain method name."""
-    test_method = "TEST"
+    test_method: str = "TESTYTEST"
```

Comment:
Was this change really necessary? Weird.

Reply:
I opted to use an uncommon (guaranteed not to be present in the docstring) value; lowering the chance of a false positive.

Reply:
Sorry, i should have been more specific—was adding `: str` necessary? I was OK with the string change, it's funny.

Reply:
Oops.  I didn't need to leave the annotation in.  
I was trying to fix a problem with mypy not liking `assert test_method in SendTESTMethod.__doc__`  because `__doc__` can be `str | None`.  

Once I added the assertion `assert SendTESTMethod.__doc__ is not None` mypy calmed down.   I just forgot to take the annotation off.